src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -1025,10 +1025,15 @@ def merge_corpus(self):
     # Hack around minijail.
     showmap_args = self._fuzz_args
     showmap_args[-1] = '1'
-    # Remove arguments for afl-fuzz.
-    self.remove_arg(showmap_args, constants.INPUT_FLAG)
+    # Remove arguments that are just for afl-fuzz.
     self.remove_arg(showmap_args, constants.DICT_FLAG)
+    self.remove_arg(showmap_args, constants.INPUT_FLAG)
```

Comment:
Fine for now. But remove_arg is ugly. I had initially because of minijail but it's not needed anymore because we don't need minijail support. I need to get rid of this.